deploy/templates/nstemplatetiers/appstudio/ns_appstudio.yaml

```diff
@@ -0,0 +1,214 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: appstudio
+objects:
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    annotations:
+      openshift.io/description: ${USERNAME}
+      openshift.io/display-name: ${USERNAME}
+      openshift.io/requester: ${USERNAME}
+    labels:
+      name: ${USERNAME}
+    name: ${USERNAME}
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: user-view
+    namespace: ${USERNAME}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: view
+  subjects:
+  - kind: User
+    name: ${USERNAME}
```

Comment:
I thought that users should have no access to the namespace at all

Reply:
Let's keep it here for now. It can be useful during development. We will need to adjust this tier anyway.